frontend/yarn.lock

```diff
@@ -0,0 +1,9393 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Could you delete this file please? Yarn is an alternative to using npm, so its kinda redundant to have both.